packages/manager/src/services/ephemeral/queries.tsx

```diff
@@ -0,0 +1,26 @@
+import { orchestratorReq } from '@app/config/orchestratorReq';
+import { useQuery } from '@tanstack/react-query';
+import { TEphemeralEnv } from './types';
+
+const ephemeralQueryKeys = {
+  list: (webPropertyName: string) => ['ephemeral-env', webPropertyName] as const
+};
+
+const path = `/webproperty/eph/list?propertyName=`;
+
+const fetchEphemeralEnvironments = async (propertyName: string): Promise<TEphemeralEnv[]> => {
+  const { data } = await orchestratorReq.get(`${path}${propertyName}`);
```

Comment:
```suggestion
  const { data } = await orchestratorReq.get(`/webproperty/eph/list?propertyName=${propertyName}`);
```

Reply:
Change added, resolving commit because the exact change has been added